.travis.yml

```diff
@@ -1,10 +1,9 @@
-language: ruby
```

Comment:
I have not tested this travis file as my application doesn't use it for CI, but I updated it for what I believe should be a working combination.